hebrew/bar/bar.md

```diff
@@ -1,4 +1,4 @@
-בר גמרה עוד לפני שנכנסת.
```

Comment:
Don't worry about this "modification". If you fix the other problems, I'll be happy to merge this.